cli/compose/convert/service.go

```diff
@@ -214,18 +214,21 @@ func convertServiceNetworks(
 		if !ok && networkName != defaultNetwork {
 			return nil, errors.Errorf("undefined network %q", networkName)
 		}
-		var aliases []string
-		if network != nil {
-			aliases = network.Aliases
-		}
 		target := namespace.Scope(networkName)
 		if networkConfig.External.External {
 			target = networkConfig.External.Name
 		}
-		nets = append(nets, swarm.NetworkAttachmentConfig{
-			Target:  target,
-			Aliases: append(aliases, name),
-		})
+		netAttachConfig := swarm.NetworkAttachmentConfig{
+			Target: target,
+		}
+		if container.NetworkMode(target).IsUserDefined() {
```

Comment:
I get that. I'm asking what the server is going to do about that. Will it be clear about the error, or will it silently ignore the aliases? Ignoring them silently is not good.

I'll just try it out myself.